WebApplication/Definitions/PublisherConfiguration.cs

```diff
@@ -38,9 +38,21 @@ public class PublisherConfiguration
         /// </summary>
         public CompletionCheck CompletionCheck { get; set; } = CompletionCheck.Polling;
 
+        private string callbackUrlBase;
+
         /// <summary>
         /// Base URL for callback.
         /// </summary>
-        public string CallbackUrlBase { get; set; }
+        public string CallbackUrlBase
+        {
+            get
+            {
+                return callbackUrlBase + "\\";
```

Comment:
What if URL already had slash?  (and should not it be `/`)?

Reply:
I think it is not an issue if there are 2 slashes. But it should be slash, not backslash.